core/server/worker/src/main/java/alluxio/web/WebInterfaceWorkerBlockInfoServlet.java

```diff
@@ -72,90 +74,96 @@ public WebInterfaceWorkerBlockInfoServlet(BlockWorker blockWorker) {
   @Override
   protected void doGet(HttpServletRequest request, HttpServletResponse response)
       throws ServletException, IOException {
-    request.setAttribute("fatalError", "");
-    String filePath = request.getParameter("path");
-    if (!(filePath == null || filePath.isEmpty())) {
-      // Display file block info
-      try {
-        UIFileInfo uiFileInfo = getUiFileInfo(new AlluxioURI(filePath));
-        List<ImmutablePair<String, List<UIFileBlockInfo>>> fileBlocksOnTier = new ArrayList<>();
-        for (Entry<String, List<UIFileBlockInfo>> e : uiFileInfo.getBlocksOnTier().entrySet()) {
-          fileBlocksOnTier.add(
-              new ImmutablePair<>(e.getKey(), e.getValue()));
+    if (Configuration.getBoolean(PropertyKey.WEB_FILE_INFO_ENABLED)) {
```

Comment:
could this return early if the file info is disabled? What behavior will users see if they access the browse page directly?

Reply:
They will see nothing. A white page with nothing on it. Will change to return earlier